### PR TITLE
the al codes are updated

### PR DESCRIPTION
**Pull request by sarismet**
I change the whole concept. Could you please review it again

### CODE REVIEW

**Comment by kayacanv** [comment]
resolve conflicts than you can merge

File: practice-app/back-end/routes/twittercomments.js [hunk 1]
```diff
@@ -11,28 +11,41 @@ router.get('/', function (req, res, next) {
   var client = new Twitter(key);
   //To specify searching parameters
   var params = {
-    q: req.query.query,
-    result_type: 'popular',
-    count: req.query.count
+    q: req.query.product_name, //get the product name
+    count: 10000,
```
[the remark this answers is not in the document]
Comment (sarismet) [reply practice-app/back-end/routes/twittercomments.js]: Done.

[the remark this answers is not in the document]
Comment (kayacanv) [reply practice-app/back-end/routes/twittercomments.js]: @sarismet  could you resolve the conflict, I think there is some spacing difference with the last pull request and yours soo

[the remark this answers is not in the document]
Comment (kayacanv) [reply practice-app/back-end/routes/twittercomments.js]: just click the resolve conflict button then delete unnecessary chars ( '>' etc. )

[the remark this answers is not in the document]
Comment (sarismet) [reply practice-app/back-end/routes/twittercomments.js]: @kayacanv I have done as you wanted.

**Pull request merged**